vassal-app/src/main/java/VASSAL/build/module/map/boardPicker/board/mapgrid/ZoneHighlight.java

```diff
@@ -187,13 +186,13 @@ public void setOpacity(int o) {
   @Override
   public String[] getAttributeDescriptions() {
     return new String[]{
-      "Name:  ",
-      "Color:  ",
-      "Coverage:  ",
-      "Width:  ",
-      "Style:  ",
-      "Image:  ",
-      "Opacity(%):  "
+      Resources.getString("Editor.ZoneHighlight.name"),
+      Resources.getString("Editor.ZoneHighlight.color"),
+      Resources.getString("Editor.ZoneHighlight.coverage"),
+      Resources.getString("Editor.ZoneHighlight.width"),
+      Resources.getString("Editor.ZoneHighlight.style"),
+      Resources.getString("Editor.ZoneHighlight.image"),
+      Resources.getString("Editor.ZoneHighlight.opacity")
```

Comment:
For standard labels, use the generic keys instead. 
"Editor.name_label"
"Editor.color_label"
"Editor.width"
I will do pass through after we are done and consolidate any outstanding duplicates, o not a huge biggy/.